src/components/license/ImageLicenseList.tsx

```diff
@@ -69,10 +67,11 @@ const ImageLicenseInfo = ({ image, locale }: ImageLicenseInfoProps) => {
       metaType: metaTypes.other,
     });
   }
+
   return (
     <MediaListItem>
       <MediaListItemImage>
-        <Image alt={image.altText} src={image.src} />
+        <img alt={image.altText} src={`${image.src}?width=200`} />
```

Comment:
Kanskje teste med litt større bilde her for å sjekke om det blir mindre kornete.
```suggestion
        <img alt={image.altText} src={`${image.src}?width=400`} />
```